config/clients/java/template/publish.gradle.mustache

```diff
@@ -32,7 +32,7 @@ publishing {
     repositories {
         maven {
             name = 'SonatypeOSSRH' // AKA: "Maven Central"
-            url = 'https://s01.oss.sonatype.org/service/local/'
+            url = 'https://s01.oss.sonatype.org/content/repositories/snapshots/'
```

Comment:
This feels off. Snapshots are typically used for internal development and use (semantically) mutable versions with a `-SNAPSHOT` suffix